net-misc/scponly/scponly-4.8-r8.ebuild

```diff
@@ -0,0 +1,249 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib readme.gentoo-r1 toolchain-funcs
+
+DESCRIPTION="A tiny pseudoshell which only permits scp and sftp"
+HOMEPAGE="https://github.com/scponly/scponly"
+SRC_URI="mirror://sourceforge/scponly/${P}.tgz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="chroot +sftp scp winscp gftp rsync unison subversion wildcards quota passwd logging"
+REQUIRED_USE="
+	|| ( sftp scp winscp rsync unison subversion )
+"
+
+RDEPEND="
+	sys-apps/sed
+	net-misc/openssh
+	chroot? ( acct-user/scponly acct-group/scponly )
+	quota? ( sys-fs/quota )
+	rsync? ( net-misc/rsync )
+	subversion? ( dev-vcs/subversion )
+	unison? ( net-misc/unison:= )
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-rsync.patch"
+	"${FILESDIR}/${P}-gcc4.4.0.patch"
+	"${FILESDIR}/${P}-sftp-server-path.patch"
+)
+
+src_configure() {
+	CFLAGS="${CFLAGS} ${LDFLAGS}" econf \
+		--with-sftp-server="/usr/$(get_libdir)/misc/sftp-server" \
+		--disable-restrictive-names \
+		$(use_enable chroot chrooted-binary) \
+		$(use_enable chroot chrooted-checkdir) \
+		$(use_enable winscp winscp-compat) \
+		$(use_enable gftp gftp-compat) \
+		$(use_enable scp scp-compat) \
+		$(use_enable sftp sftp) \
+		$(use_enable quota quota-compat) \
+		$(use_enable passwd passwd-compat) \
+		$(use_enable rsync rsync-compat) \
+		$(use_enable unison unison-compat) \
+		$(use_enable subversion svn-compat) \
+		$(use_enable subversion svnserv-compat) \
+		$(use_enable logging sftp-logging-compat) \
+		$(use_enable wildcards wildcards)
+}
+
+src_compile() {
+	emake CC=$(tc-getCC)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+
+	dodoc AUTHOR BUILDING-JAILS.TXT CHANGELOG CONTRIB README SECURITY TODO
+
+	if use chroot ; then
+
+		DOC_CONTENTS="
+	You might want to run\n
+	emerge --config =${CATEGORY}/${PF}\n
+	\nto setup the chroot. Otherwise you will have to setup chroot manually
+	Please read the docs in /usr/share/doc/${PF} for more informations, also
+	the SECURITY file.
+"
```

Comment:
If you move it from global scope into function scope, you could make the variable `local` as well.

Indentation looks strange.

Reply:
Will do.  It _is_ strange.  I'll look at other cases in the tree where DOC_CONTENTS is used conditionally based on an 'if use ...' test.  I only see two at the moment, media-gfx/jpeg2ps/jpeg2ps-1.9-r2.ebuild and gnome-base/nautilus/nautilus-3.32.3.ebuild.